tests/linux_benchmarks/netperf_benchmark_test.py

```diff
@@ -105,19 +105,7 @@ def testExternalAndInternal(self):
          ('TCP_CRR_Latency_max', 2500.0, 'us'),
          ('TCP_CRR_Latency_stddev', 551.07, 'us'),
          ('TCP_STREAM_Throughput', 1187.94, mbps),
-         ('TCP_STREAM_Latency_p50', 2.0, 'us'),
```

Comment:
Ah - I see why the `if unit != MBPS` was included above now. I was confused why TCP_STREAM had latency stats, but if you look in the output file that we read this data from, the stats are there. Do we know whether a human or a machine made that file?